### DISCUSSION TITLE
Tried it and WAS NOT IMPRESSED, am I doing it wrong??

### DISCUSSION THREAD

**Discussion by GPU-server** [SWivid/F5-TTS#226] [Q&A]
I downloaded the sstuff and installed libraries.
run gradio. opened the web server.
I went to inference and inserted a video game character audio (without music, about 20 seconds) and then written a text for it to generate it.
First seconds generated had a word that was invented, then the rest (the actual text I gave it) was not that good quality (noise, or not calibratied, not good quality)

Why is everyone talking about F5 TTS, I surely configured it badly or I am missing something? Please someone one tell me. I wanna be impressed.

**Comment by SWivid** [comment]
#225 https://voca.ro/13QHMeIsGs0e
Our project is completely free and open source, so maybe you can try to fine-tune it on the out-of-trainset data or pay a commercial company to do it for you, or switch to a model that you're happy with.
Anyway, have fun.

**Reply by GPU-server** [reply]
Thanks for the message. Here it was not a critize more like, a call for help you know.
This voice looks like "Jesse" from Breakind Bad haha. But I did not play all the games, I think I recognize it but not sure.
I remember using another TTS voice where we "train" voices, maybe I should do that with T5-TTS and gain more results? 
I will check youtube and hope to get more help. I wish there was like an example "that always work" that shows all the capabilities.
I see for example there is a "podcast" tab, but did not even try it by fear .. not knowing at all how it works.
So please forgive my post, it is not just to criticize, it was more like "a call" for guidance, on how to make this work the best ^^. Thanks again

**Comment by NehaB18** [comment]
@GPU-server in my experience reference voice sample +  actual text to generate voice should be less than 30 seconds for it to work properly. So in my case 4/5 seconds reference voice sample + actual text to generate voice (20 seconds) worked fine.
